dom5.ts

```diff
@@ -403,6 +403,17 @@ export function nodeWalkPrior(node: Node, predicate: Predicate): Node|
   return undefined;
 }
 
+export function nodeWalkParent(node: Node, predicate: Predicate): Node|undefined {
```

Comment:
Changed.  Also added comment for the method and demonstrated in test that the link is not a direct ancestor.